helm/harvester/charts/harvester/sandbox/lsst.init-harvester

```diff
@@ -0,0 +1,14 @@
+# install CA files
```

Comment:
I think you can remove L1-L7. The same for lsst.run-harvester-crons.

Reply:
Sorry. I forgot to remove them after changing the init-script and run-harvester-crons.
removed.